Cargo.toml

```diff
@@ -103,7 +105,7 @@ cookie_crate = { version = "0.12", package = "cookie", optional = true }
 cookie_store = { version = "0.10", optional = true }
 
 ## gzip
-async-compression = { version = "0.2.0", default-features = false, features = ["gzip", "stream"], optional = true }
+async-compression = { version = "0.2.0", default-features = false, features = ["gzip", "brotli", "stream"], optional = true }
```

Comment:
This will mean brotli and gzip are both enabled even if only 1 is desired. It should be possible to instead write them like this:


```toml
# features

gzip = ["async-compression", "async-compression/gzip"]
brotli = ["async-compression", "async-compression/brotli"]


# dependencies

async-compression = { version = "0.2.0", default-features = false, features = ["stream"], optional = true }
```